docs/sidebars.js

```diff
@@ -9,12 +9,15 @@ module.exports = {
   'docs': {
     'Introduction': [
       'introduction/overview',
+      'introduction/getting-started',
       'introduction/community'
     ],
-    'Architecture': [
-      'architecture/overview'
+    'Concepts': [
+      'concepts/overview',
+      'concepts/package-arch'
     ],
     'Packages': [
+      'packages/creating-packages',
```

Comment:
I wonder if this shouldn't rather be within the concepts section? this relates to how to create more packages which could be seen as a concept which can be approached in a few ways. The packages section contains all the currently built packages for the community to make use of

Reply:
Yeah I was wondering about that to, was also considering a How-tos section, but didn't think we had enough to go in there. Cool, let me move it to concepts and see how it feels.